system/jlib/jsmartsock.ipp

```diff
@@ -104,6 +105,7 @@ public:
     virtual StringBuffer & getUrlStr(StringBuffer &str, bool useHostName);
     virtual bool isTlsService() const override { return tlsService; }
     virtual const IPropertyTree *queryTlsConfig() const { return tlsConfig; };
+    const char *queryTlsIssuer(){return issuer.str();}
```

Comment:
trivial: could be const and missing spaces compared with above and normal.

Reply:
Updated.